### ISSUE TITLE
Resnet 18 learning rate seems unreasonably HIGH; might prevent convergence

### ISSUE THREAD

**Issue by paplhjak** [pwhofman/probly#445]
https://github.com/pwhofman/probly/blob/b3a3f641d0fa641dc6505db0a0132ec954eac6ee/src/probly_benchmark/configs/recipe/resnet18_cifar10.yaml#L15

Wouldnt LR of 0.001 or 0.003 make more sense as a starting point?

**Comment by pwhofman** [comment]
The hyper-parameters are mainly based upon https://github.com/kuangliu/pytorch-cifar. Getting quite a good accuracy (~95%). Do you have a reference to or results for hyper-parameters that give better performance?
